dmf/localization/dmf.lua

```diff
@@ -32,13 +32,11 @@ return {
   percent = {
     en = "%%",
   },
-  toggle_mod = {
```

Comment:
Revert these localization changes

Reply:
Well I added `toggle_mods` for the toggle category name and description, which is for toggling more than one mod. Then the old `toggle_mod` wasn't used anywhere anymore, so I removed it. But I'm happy to make changes / revert if that's preferred.

Reply:
Ah I see now - missed the s